test/DynamoCoreWpfTests/CoreUITests.cs

```diff
@@ -511,25 +513,27 @@ public void PreferenceSetting()
 
             initalSetting.ConnectorType = ConnectorType.BEZIER;
             initalSetting.ConsoleHeight = 100;
-            initalSetting.IsBackgroundPreviewActive = true;
+            initalSetting.SetIsBackgroundPreviewActive(backgroundPreviewName, true);
 
             initalSetting.Save(tempPath);
```

Comment:
all tests related to preferences are modified to read/write `IsBackgroundPreviewActive` value in the new way